Objects/Converters/ConverterAutocadCivil/ConverterAutocad2021/ConverterAutocad2021.csproj

```diff
@@ -29,6 +29,8 @@
   </Target>
 
   <ItemGroup>
+    <PackageReference Include="WindowsBase_Core.STW" Version="4.0.30319.1" />
+    <PackageReference Include="EPP.PresentationCore" Version="1.0.0" />
```

Comment:
Are these needed in the converters?
Even if they were, we should probably be using the `System.Windows` ones, no?

Reply:
Maybe, but I didn’t get. The purpose of this PR is a initiative of converting CAD texts to speckle. It’s a good thing not merge with Main or Dev yet…. I think so

Reply:
It’s working in a reasonable way.

Reply:
Ok! Thanks, Pedro!
Our main concern is around performance implications, we will do some tests to see how it might affect sending, if it's not significant we'll be happy to include this in.

Let us know if you have done any tests already.

In the future, ideally, the viewer will be able to natively display text without us converting it to lines :)

Reply:
Good news! I also think the right way to do this is to make the viewer able to display texts natively (using normal vector, box, font and text).

Reply:
I did few tests, so I don't have benchmarks of it. But I think the main pain of this approach is the connector processing. Perhaps this can slow down the process.